Floofbot/Services/WordFilterService.cs

```diff
@@ -74,7 +72,7 @@ class WordFilterService
             {
                 Regex r = new Regex(@$"\b({filteredWord.Word})\b",
                     RegexOptions.IgnoreCase | RegexOptions.Singleline);
-                if (r.IsMatch(messageContent))
+                if (r.IsMatch(Regex.Escape(messageContent)))
```

Comment:
If I'm reading this right, you're escaping regex on the message content, not on the filtered word. I assume it should be `Regex.Escape(filteredWord.Word)` instead?

Reply:
yes

Reply:
Resolved